src/obkrnl/src/malloc/stage2.rs

```diff
@@ -54,7 +56,14 @@ impl Stage2 {
             zones
         });
 
-        Self { zones }
+        // TODO: Is there a better way than this?
```

Comment:
std::vec::from_elem

Reply:
But that's only for Clone types

Reply:
Not sure where did you get this from since I can't find this method on both `std` and `alloc`.

Reply:
> But that's only for types that implement Clone

This is a problem since `Stats` cannot be cloned due to atomic fields.